demo/pkg/subgraphs/employees/subgraph/schema.graphqls

```diff
@@ -55,33 +64,41 @@ enum OperationType {
 
 type Operator implements RoleType {
   departments: [Department!]!
-  operatorType: [OperationType!]!
   title: [String!]!
+  employees: [Employee!]! @goField(forceResolver: true)
+  operatorType: [OperationType!]!
+}
+
+type Details @key(fields: "id") {
```

Comment:
Could we not make Details an entity?

It illustrates sharable objects, sharable planning is quite different from regular entity planning